src/www/app/app.ts

```diff
@@ -41,17 +46,30 @@ export function unwrapInvite(s: string): string {
       //  - When a user opens invite.html#ENCODEDSSURL in their browser, the website (currently)
       //    redirects to invite.html#/en/invite/ENCODEDSSURL. Since copying that redirected URL
       //    seems like a reasonable thing to do, let's support those URLs too.
-      const possibleShadowsocksUrl = decodedFragment.substring(decodedFragment.indexOf('ss://'));
+      //  - Dynamic keys are not supported by the invite flow, so we don't need to check for them
+      const possibleShadowsocksUrl = decodedFragment.substring(decodedFragment.indexOf(`${OUTLINE_PROTOCOL.STATIC}//`));
 
-      if (new URL(possibleShadowsocksUrl).protocol === 'ss:') {
+      if (new URL(possibleShadowsocksUrl).protocol === OUTLINE_PROTOCOL.STATIC) {
         return possibleShadowsocksUrl;
       }
     }
   } catch (e) {
-    // Something wasn't a URL, or it couldn't be decoded - no problem, people put all kinds of
-    // unexpected things in the clipboard.
+    // It wasn't an invite URL!
   }
-  return s;
+
+  return possiblyInviteUrl;
+}
+
+// Returns true if the given url was a valid Outline invitation or
+// access key
+export function isOutlineServiceLocation(url: string): boolean {
```

Comment:
Let's use `isOutlineAccessKey` to reuse the product concept. Move the unwrap invite out.
You need to update the tests as well.